src/Symfony/Component/Validator/Tests/Constraints/TimezoneValidatorTest.php

```diff
@@ -73,7 +73,6 @@ public function getValidTimezones(): iterable
         yield ['EST5EDT'];
         yield ['MST7MDT'];
         yield ['PST8PDT'];
-        yield ['America/Montreal'];
```

Comment:
This does not seem to be related to the problem described in the issue. Can you revert this change?